uPortal-webapp/src/main/webapp/media/skins/respondr/view/less/activity.less

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
:thinking: It could, the reason I went for `../skins/respondr/view/less` is because these styles are not common, they are specific to a single view.